generate/workers.py

```diff
@@ -455,24 +455,23 @@ def aws(
     return wp
 
 
-@lru_cache(maxsize=2)
-def azure_machine_types_by_location():
+@lru_cache(maxsize=100)
+def azure_machine_types_in_location(location):
     """
     Return the set of machine types (such as "Standard_F32s_v2") in an Azure location.
 
       location: The Azure location, such as "centralus"
 
-    The instances are read from config/azure-machine-type-offerings.json and
+    The instances are read from JSON files in config/ec2-instance-type-offerings, and
```

Comment:
this path is confusing?

```suggestion
    The instances are read from JSON files in config/azure-machine-type-offerings, and
```

Reply:
eagle eye!!! amazing, thanks!